estate/models/estate_property.py

```diff
@@ -0,0 +1,96 @@
+from odoo import api, models, fields
+from odoo.exceptions import UserError
+import datetime
+from dateutil.relativedelta import relativedelta
+from odoo.tools.float_utils import float_compare, float_is_zero
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Property for the Real Estate app"
```

Comment:
```suggestion
    _description = "Estate Property"
```
_description contains a short human-readable name for the model